scripts/upgrade.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+# usage: TO=v0.9.18 scripts/upgrade.sh
+
+# The following line ensure we run from the project root
+PROJECT_ROOT=`git rev-parse --show-toplevel`
+cd ${PROJECT_ROOT}
+
+
+FROM=`grep "^substrate-build-script-utils" ./node/parallel/Cargo.toml | egrep -o "(v[0-9\.]+)"`
+TO=${TO}
+
+cargo_toml_list=$(find . -name "Cargo.toml")

```

Comment:
should also ignore target directory

Reply:
yes, target dir and Cargo.lock should ignore